lib/index.rb

```diff
@@ -1,20 +1,56 @@
+# !/usr/bin/env ruby -I ../lib -I lib
 # frozen_string_literal: true
-#!/usr/bin/env ruby -I ../lib -I lib
-# coding: utf-8
 
 require 'sinatra'
 require './lib/file_customer_gateway'
 require './lib/use_cases/save_customer_details'
+require './lib/use_cases/view_summary'
 require './lib/builder/customer'
 require './lib/domain/address'
 require './lib/domain/customer'
 
+class ItemsGateway
+  def all
+    [
+      { part_id: 456, part_name: 'Drones', part_price: 144, part_quantity: 34 },
```

Comment:
part_id needs to be a string to match the data from the add_items use case.
Also, based on Craig's input, we will be changing our part_price and part_quantity to strings also.